### PR TITLE
fix: bump wrangler to 2.x and fix build

### PR DESCRIPTION
**Pull request by ksco**
Some fixes are related to macOS, some may be related to Wrangler version, I submitted them together.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -45,7 +45,7 @@ $(BUILD_DIR)/index.html.liquid: frontend/index.html frontend/index.js frontend/s
 $(all_html): $(BUILD_DIR)/%.html: $(BUILD_DIR)/%.html.liquid $(CONF) $(html_renderer)
 	node $(html_renderer) -c $(CONF)  -o $@ $<
 	@# remove indents to reduce size
-	sed -E -i 's/^\s+//g' $@
+	sed -E -i '' 's/^\s+//g' $@
```
Comment (SharzyL) [inline Makefile:48]: For FreeBSD sed, it is not recommended to use a zero-length extension, use a meaningful extension (like `.backup` instead.

File: scripts/md2html.sh [hunk 2]
```diff
@@ -15,12 +15,12 @@ cat > "$html_file" <<-EOF
...
 </head>
 <div class="container-lg px-3 my-5 markdown-body">
-</html>
 EOF
 
 yarn -s remark "$md_file" --use remark-html >> "$html_file"
 
 cat >> "$html_file" <<-EOF
 </div>
 </body>
```
Comment (SharzyL) [inline scripts/md2html.sh:24]: Sorry for my carelessness but there is also a missing opening `<body>` tag

**Review by SharzyL — CHANGES_REQUESTED** [review]

File: scripts/deploy-static.sh [hunk 3]
```diff
@@ -7,11 +7,14 @@ declare -a args=('yarn' 'wrangler' 'kv:key' 'put' '--binding=PB')
 if [ "$1" == '--preview' ]; then
 	args+=('--preview')
 	shift
+else
+	args+=('--preview')
+	args+=('false')
```
[the remark this answers is not in the document]
Comment (ksco) [reply scripts/deploy-static.sh:12]: Sorry, I accidentally installed a new version of wrangler.

File: scripts/md2html.sh (hunk 2 above)
Comment (ksco) [reply scripts/md2html.sh:25]: Fixed.

File: Makefile (hunk 1 above)
Comment (ksco) [reply Makefile]: Fixed, I used perl instead, it's more portable.

**Comment by ksco** [comment]
I upgraded wrangler in 542ebab, 1.19.0 have many issues at least on my machine.

**Comment by SharzyL** [comment]
Thanks for your PR. Since wrangler 2.x is incompatible with 1.x in many ways, I will make a few more changes to the build system.

**Pull request merged**